static-analysis/autodispose-error-prone/src/test/java/com/uber/autodispose/errorprone/UseAutoDisposeTest.java

```diff
@@ -32,43 +32,68 @@
 
   private CompilationTestHelper compilationHelper;
 
-  @Before public void setup() {
+  @Before
+  public void setup() {
     compilationHelper = CompilationTestHelper.newInstance(UseAutoDispose.class, getClass());
   }
 
-  @Test public void test_autodisposePositiveCasesWithDefaultClass() {
+  @Test
+  public void test_autodisposePositiveCasesWithDefaultClass() {
     compilationHelper.addSourceFile("UseAutoDisposeDefaultClassPositiveCases.java").doTest();
   }
 
-  @Test public void test_autodisposePositiveCasesWithDefaultClassGivenCustomTypes() {
-    compilationHelper.setArgs(ImmutableList.of("-XepOpt:TypesWithScope"
-        + "=com.uber.autodispose.errorprone.ComponentWithLifecycle"));
-    compilationHelper.addSourceFile("UseAutoDisposeDefaultClassPositiveCases.java").doTest();
+  @Test
```

Comment:
Why the annotation on separate line? All other tests have it on the same line right?

Reply:
Going to start slowly transitioning this to GJF, started with this